configuration/lambda_config.template

```diff
@@ -11,10 +11,13 @@
       "us-east-1"
     ]
   },
-  "config_cache_ttl": $EXODUS_CONFIG_CACHE_TTL,
+  "config_cache_ttl": "$EXODUS_CONFIG_CACHE_TTL",
```

Comment:
Was there a reason you wanted to wrap this and the next one in quotes?  The values to be substituted are integers.  It doesn't seem necessary to wrap them in strings just to unwrap them again later.

I think we've been through this before, there was confusion around this file not being valid JSON (which it is not if there's no quotes around the $ placeholders).  But that's why it was renamed to .template to more accurately reflect that it's not a JSON file, it's a template which can be processed in order to generate a JSON file.  If you had to add the quotes to make something work, there was probably a bug somewhere else.

Reply:
Yeah, I think there was a bug somewhere in the testing setup but it seems to be resolved now.